packages/angular_devkit/build_optimizer/src/build-optimizer/build-optimizer.ts

```diff
@@ -70,9 +79,10 @@ export function buildOptimizer(options: BuildOptimizerOptions): TransformJavascr
 
   if (inputFilePath
```

Comment:
No idea where this path comes form, but is it guaranteed to have a `/` before `node_modules`?
E.g. is there any chance it might start with `node_modules/@angular/...`?

Reply:
It is guaranteed (in our usage) to be a full, non-relative path.